### DISCUSSION TITLE
Glob patterns in Zettlr 2.0

### DISCUSSION THREAD

**Discussion by hildebrand-institute** [Zettlr/Zettlr#2449] [Q&A] [answered]
Hi Hendrik,
If I got it right "glob patterns" should be used for indicating a sequence of specific files, among all those included into the project directory, to be exported. Can you please give me an example of how to write a glob pattern? Writing the simple sequence of the files' names does not work...
Thanks

**Comment by nathanlesage** [comment, marked as answer]
If you do not specify any glob pattern **all** files will simply be exported. The glob patterns act as filters, so as soon as you specify at least one, all files will be checked against this.

Glob patterns can do quite a lot of stuff, so it's best to just have a look online for tutorials, for example [here](https://www.malikbrowne.com/blog/a-beginners-guide-glob-patterns).

(Please note that sometimes, glob patterns work differently across implementations, so if you're in doubt, have a look at the documentation of Zettlr's specific implementation, [minimatch](https://github.com/isaacs/minimatch).)

**Reply by hildebrand-institute** [reply]
Very nice sorting method, it works like a charm. Thanks
